common/common-rest/src/main/java/org/apache/servicecomb/common/rest/AbstractRestInvocation.java

```diff
@@ -142,6 +145,18 @@ protected void scheduleInvocation() {
     });
   }
 
+  private boolean isInQueueTimeout() {
+    Object timeout = requestEx.getAttribute(RestConst.REST_REQUEST_IN_QUEUE_TIME);
```

Comment:
1.direct use invocation.getStartTime()?
is's a nanotime.

2.when invoke from highway, need this logic also